app/task_routes.py

```diff
@@ -0,0 +1,86 @@
+from app import db
+from .models.task import Task
+from .models.goal import Goal
+from flask import Blueprint, request, make_response, jsonify, abort
+import sqlalchemy 
+from .route_helpers import validate_model_id
+
+
+
+task_bp = Blueprint("task_bp", __name__, url_prefix="/tasks")
+
+
+# all task methods
+    # Create
+@task_bp.route("", methods=["POST"])
+def create_task():
+    
+    request_body = request.get_json()
+    new_task = Task.from_dict(request_body)
```

Comment:
There doesn't seem to be any validation to check to see if the request body has a title or description.